src/Elan.Web/ClientApp/src/api/PostsApi.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+
+export const savePost = async (content) => {
+    var api = "/api/Posts/CreatePost";
+
+    return await axios.post(api,
+        {
+            content: content
+        }, 
+        {
+            headers: {
+                Authorization: 'Bearer ' + localStorage.getItem('token')
```

Comment:
Could you configure axios so it will automatically attach this header on every request?

Reply:
Yes, in the next week